Services/Interfaces/web_interface/api/metadata.py

```diff
@@ -34,6 +34,13 @@ def user_feedback():
         catch_exception=True))
 
 
+@api.api.route("/user_suggest")
+def user_suggest():
+    return json.dumps(external_resources_manager.get_external_resource(
```

Comment:
I think this should be a constant in OctoBot like other links instead of an external resource

Reply:
You're right :+1: